document-portal/meson.build

```diff
@@ -0,0 +1,102 @@
+built_src_permission_store = gnome.gdbus_codegen('permission-store-dbus',
+    sources: '../data/org.freedesktop.impl.portal.PermissionStore.xml',
+    interface_prefix: 'org.freedesktop.impl.portal',
+    namespace: 'Xdg',
+    autocleanup: 'none',
+)
+
+src_db = files(
+    'permission-db.c',
+    'permission-db.h',
+    'gvdb/gvdb-reader.h',
+    'gvdb/gvdb-format.h',
+    'gvdb/gvdb-reader.c',
+    'gvdb/gvdb-builder.h',
+    'gvdb/gvdb-builder.c',
+)
+
+src_xdg_permission_store = [
```

Comment:
```meson
src_xdg_permission_store = [
  files(...),
  ....,
]

Reply:
done, without the `files()` bit given @eli-schwartz comment below